### PR TITLE
stat/distuv: provide complete InverseGamma test coverage

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Please take a look.

### CODE REVIEW

File: stat/distuv/inversegamma.go [hunk 1]
```diff
@@ -35,9 +35,6 @@ func (g InverseGamma) CDF(x float64) float64 {
 	if x < 0 {
 		return 0
 	}
-	// TODO(btracey): Replace this with a direct call to the upper regularized
-	// gamma function if mathext gets it.
-	//return 1 - mathext.GammaInc(g.Alpha, g.Beta/x)
```
Comment (kortschak) [inline stat/distuv/inversegamma.go:40]: @btracey This was as it was when the file was originally added. Which do you want, `1 - mathext.GammaIncReg(g.Alpha, g.Beta/x)` or `mathext.GammaIncRegComp(g.Alpha, g.Beta/x)` since both are available.

This should be a separate PR though.

Comment (romanwerpachowski) [reply stat/distuv/inversegamma.go:40]: I'll revert it and let's fix it in a separate PR.

**Comment by romanwerpachowski** [comment]
PTAL.
